editor/src/messages/tool/tool_messages/select_tool.rs

```diff
@@ -1,12 +1,15 @@
+use std::fmt;
```

Comment:
Should be moved down into the third group of imports (the third-party group, together with `glam` and `serde`). Or, since it's short, just remove this import and fully qualify its usages.